dspace-api/src/test/java/org/dspace/discovery/DiscoveryTest.java

```diff
@@ -0,0 +1,124 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.discovery;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.sql.SQLException;
+import java.util.List;
+
+import org.apache.logging.log4j.Logger;
+import org.dspace.AbstractIntegrationTest;
+import org.dspace.authorize.AuthorizeException;
+import org.dspace.content.Collection;
+import org.dspace.content.Community;
+import org.dspace.content.WorkspaceItem;
+import org.dspace.content.factory.ContentServiceFactory;
+import org.dspace.content.service.CollectionService;
+import org.dspace.content.service.CommunityService;
+import org.dspace.content.service.InstallItemService;
+import org.dspace.content.service.ItemService;
+import org.dspace.content.service.WorkspaceItemService;
+import org.dspace.services.factory.DSpaceServicesFactory;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * This class will aim to test Discovery related use cases
+ */
+public class DiscoveryTest extends AbstractIntegrationTest {
```

Comment:
One other thing I just realized.  As this is an *IntegrationTest*, it should have `IT` in the name...otherwise, it will be run alongside *unit tests* instead of Integration Tests.  So, this should be renamed something like `DiscoveryIT` or `ITDiscovery`.  

Also, in the `destroy()` method you should be manually cleaning up any objects created in the `init()` method.
For examples, see other test classes that specify `extends AbstractIntegrationTest`